base/sparse/sparsevector.jl

```diff
@@ -1305,8 +1303,14 @@ function minimum{T<:Real}(x::AbstractSparseVector{T})
      min(zero(T), minimum(nonzeros(x))))::T
 end
 
-maxabs{T<:Number}(x::AbstractSparseVector{T}) = maxabs(nonzeros(x))
-minabs{T<:Number}(x::AbstractSparseVector{T}) = nnz(x) < length(x) ? abs(zero(T)) : minabs(nonzeros(x))
+for f in [:sum, :maximum, :minimum], op in [:abs, :abs2]
```

Comment:
I think this is fine for now since it covers what we already have but I was wondering if we could get something more general (probably not in this PR) with something like
```jl
mapreduce(f, op, x::AbstractSparseVector) = op(mapreduce(f, op, nonzeros(x)), repeat(op, f(zero(x[1])), countnz(x)))
```
where the `repeat` functions could get specialized, e.g.
```jl
repeat(::typeof(+) , x, n::Integer) = x*n
```
There are probably problems with this approach that I haven't seen yet but it seems nice that it can handle arbitrary return values of the zero input and be fast for the important cases.